F/FLINT/build_tarballs.jl

```diff
@@ -56,6 +64,9 @@ make install
 # platforms are passed in on the command line
 platforms = supported_platforms()
 
+# Currently skipped due to the following error: `configure: error: Could not find mpfr.h`
+filter!(p -> arch(p) != "riscv64", platforms)
```

Comment:
I guess we could retry once https://github.com/JuliaPackaging/Yggdrasil/pull/10637 is merged -- but that isn't a priority for anyone given that there is no Julia for RISC V yet